### DISCUSSION TITLE
UMD-files for script-with-src usage?

### DISCUSSION THREAD

**Discussion by quite** [dimsemenov/PhotoSwipe#1932] [General]
I'm using photoswipe 4 by loading the minimized js-files using a script-with-src-tag, and then some more js-code in another script tag. I prefer this way of adding minimal amout of js to an otherwise static site built from plain markdown files. If I migrate to version 5, is it the UMD files that I must use to achieve this?

**Comment by dimsemenov** [comment]
Ideally, use ES modules and include the library via `<script type="module">`, if for whatever reason you cannot do this  - use transpiled UMD version in `dist/umd/` folder of the repository.

**Reply by quite** [reply]
Aha, so the following from the docs should work. The imports of `photoswipe/…` will work if I drop those files at the correct path in my web root?

```
<script type="module">
import PhotoSwipeLightbox from 'photoswipe/dist/photoswipe-lightbox.esm.js';
const lightbox = new PhotoSwipeLightbox({
  gallery: '#my-gallery',
  children: 'a',
  pswpModule: () => import('photoswipe/dist/photoswipe.esm.js')
});
lightbox.init();
</script>
```

**Reply by quite** [reply]
OK, it works, tried. I'm a js noob. Thanks!
